src/instructlab/model/dk_bench_utils.py

```diff
@@ -0,0 +1,320 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=ungrouped-imports
+# Standard
+from datetime import datetime
+from typing import List
+import enum
+import logging
+import os
+import pathlib
+
+# Third Party
+from instructlab.eval.mt_bench_common import (
+    get_openai_client as get_local_openai_client,
+)
+from instructlab.eval.ragas import ModelConfig, RagasEvaluator
+from ragas.evaluation import EvaluationResult  # type: ignore
+import click
```

Comment:
can we avoid using click outside of the `src/instructlab/cli` package? This is tech debt we will have to fix in #1548